contrib/ci/test.sh

```diff
@@ -32,9 +32,9 @@ echo "1234567890123456789012345678901234567890" > $CURRENT_DIR/tmp/aws-admin/acc
 # create fake AWS credentials for minio user credentials
 mkdir -p $CURRENT_DIR/tmp/aws-user
 # needs to be 20 characters long
-echo "12345678901234567890" > $CURRENT_DIR/tmp/aws-user/access-key-id
+echo "12345678901234567890" > $CURRENT_DIR/tmp/aws-user/accesskey
```

Comment:
these changes broke minio. It seems minio reads in `/var/run/secrets/deis/minio/user/access-key-id` instead of `accesskey`. Would you please update the test script so it'll boot up for minio?